### PR TITLE
WebAPI: Update Method pages to modern structure (part 15)

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Continuing #14857 

## Summary
Updating the method pages to follow the current templates:
https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections#constructors_and_methods
https://developer.mozilla.org/en-US/docs/MDN/Structures/Page_types/API_method_subpage_template

Changes Include:
- fix syntax section
- changing section titles to suggested titles in the template
- change `{{DOMxRef("DOMString")}}` to `string`
- remove extra spaces in between words and extra trailing spaces
- other small corrections

### Metadata
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/speechgrammarlist/addfromstring/index.md [hunk 1]
```diff
@@ -17,23 +17,23 @@ browser-compat: api.SpeechGrammarList.addFromString
...
 {{domxref("SpeechGrammarList")}} interface takes a grammar present in a specific
-{{domxref("DOMString")}} within the code base (e.g. stored in a variable) and adds it to
+string within the code base (e.g. stored in a variable) and adds it to
 the `SpeechGrammarList` as a new {{domxref("SpeechGrammar")}} object.
 
 ## Syntax
 
 ```js
-speechGrammarListInstance.addFromString(string,weight);
+addFromString(string,weight)
```
Comment (teoli2003) [inline files/en-us/web/api/speechgrammarlist/addfromstring/index.md:26]:
```suggestion
addFromString(string)
addFromString(string, weight)
```

File: files/en-us/web/api/speechgrammarlist/addfromuri/index.md [hunk 2]
```diff
@@ -26,17 +26,17 @@ specified by URI.
 ## Syntax
 
 ```js
-speechGrammarListInstance.addFromURI(src,weight);
+addFromURI(src,weight)
```
Comment (teoli2003) [inline files/en-us/web/api/speechgrammarlist/addfromuri/index.md:29]:
```suggestion
addFromURI(src)
addFromURI(src, weight)
```

File: files/en-us/web/api/speechgrammarlist/addfromuri/index.md [hunk 3]
```diff
@@ -26,17 +26,17 @@ specified by URI.
...
 ```
 
-### Returns
+### Return value
```
Comment (teoli2003) [inline files/en-us/web/api/speechgrammarlist/addfromuri/index.md:32]: This section must be moved after the _Parameters_ section.

File: files/en-us/web/api/svggeometryelement/getpointatlength/index.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ point at a given distance along the path.
 ## Syntax
 
 ```js
-DOMPoint someElement.getPointAtLength(float distance);
+getPointAtLength(float distance)
```
Comment (teoli2003) [inline files/en-us/web/api/svggeometryelement/getpointatlength/index.md:23]:
```suggestion
getPointAtLength(distance)
```

File: files/en-us/web/api/svgpointlist/insertitembefore/index.md [hunk 5]
```diff
@@ -16,9 +16,11 @@ The **`insertItemBefore()`** method of the {{domxref("SVGPointList")}} interface
 ## Syntax
 
 ```js
-SVGPointList.insertItemBefore(obj,index);
+insertItemBefore(obj,index)
```
Comment (teoli2003) [inline files/en-us/web/api/svgpointlist/insertitembefore/index.md:19]:
```suggestion
insertItemBefore(obj, index)
```

File: files/en-us/web/api/svgpointlist/replaceitem/index.md [hunk 6]
```diff
@@ -16,9 +16,11 @@ The **`replaceItem()`** method of the {{domxref("SVGPointList")}} interface repl
 ## Syntax
 
 ```js
-SVGPointList.replaceItem(obj,index);
+replaceItem(obj,index)
```
Comment (teoli2003) [inline files/en-us/web/api/svgpointlist/replaceitem/index.md:19]:
```suggestion
replaceItem(obj, index)
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Comment by OnkarRuikar** [comment]
@teoli2003 done

**Review by teoli2003 — APPROVED** [review]
Thanks!

**Pull request merged**